src/index.html

```diff
@@ -11,7 +11,189 @@
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agree"
+            id="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="registration">
+        <legend>Registration</legend>
+        <div class="registrationItem email">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div class="registrationItem password">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="facts">
+        <legend>An interesting fact about you!</legend>
+        <div class="love">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="yesOrNo"
+            value="y"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="yesOrNo"
+            value="n"
+            >
+            No
+          </label>
+        </div>
+        <div class="favColor">
+          <label for="colorPalette">
+            What is your favorite color?
+            <input
+            type="color"
+            name="color"
+            id="colorPalette"
+            >
+          </label>
+        </div>
+        <div class="date">
+          <label for="dateTime">
+            What time do you go to bed?
+          </label>
+          <input
+          id="dateTime"
+          type="time"
+          name="dateTime"
+          step="2"
+          >
+        </div>
+
+        <div class="cars">
+          <label for="carsSelection">
+            What are your favorite brand of cars?
+            <select
+              id="cars"
+              name="cars"
+              size="4"
+              multiple
+            >
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="rating">
+          <label for="ranging">
+            How do you rate our work?
+            <input
+            type="range"
+            id="ranging"
+            value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="addInfo">
+        <legend>Additional info</legend>
+        <div class="comments">
+          Comments:
+          <textarea name="comment"
+          id="comment"
+          cols="25"
+          rows="2"
+          minlength="10"
+          maxlength="99"
+          autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea name="comment"
            id="comment"
            cols="25"
            rows="2"
            minlength="10"
            maxlength="99"
            autocomplete="off"
        ></textarea>
```


https://user-images.githubusercontent.com/102413088/192488163-2695cb58-042e-41c2-bc6b-6d37f55a2d25.mov

виправ поведінку textarea

https://mate.academy/learn/css-advanced/forms#/video/885

Reply:
Така поведінка виправляється лише якщо відкриваючий і замикаючий ">" у одній лінії :(